express_relay/sdk/js/src/index.ts

```diff
@@ -99,18 +104,55 @@ export type OpportunityBid = {
   bid: BidInfo;
 };
 
+/**
+ * Represents a raw bid on acquiring a permission key
+ */
+export type Bid = {
```

Comment:
this is not quite the same type unfortunately. there is a `snake_case` -> `camelCase` conversion and I chose types that made more sense in a js sdk instead of the general openapi types. I'm not sure if there is an easy way to apply these changes on top of the openapi type.